sup3r/pipeline/forward_pass.py

```diff
@@ -770,17 +727,28 @@ def __init__(self, file_paths, model_kwargs, fwp_chunk_shape,
         self.s_enhance = np.product(self.s_enhancements)
         self.t_enhance = np.product(self.t_enhancements)
 
-        self.fwp_slicer = ForwardPassSlicer(self.grid_shape,
-                                            self.raw_time_index,
+        self.fwp_slicer = ForwardPassSlicer(self.grid_shape, self.raw_tsteps,
                                             self.temporal_slice,
                                             self.fwp_chunk_shape,
                                             self.s_enhancements,
                                             self.t_enhancements,
                                             self.spatial_pad,
                                             self.temporal_pad)
 
-        logger.info('Initializing ForwardPassStrategy for '
-                    f'{self.input_file_info}. Using n_nodes={self.nodes} with '
+        self.preflight()
+
+    @property
+    def worker_attrs(self):
```

Comment:
This seems like a bad use of a property. Properties enable encapsulated compute. Whats the reason to have a hard-coded property value?

Reply:
Yeah no reason. I guess better as a method?

Reply:
No no my point was just make it an attribute! self._protected attribute if you don't want it messed with. Doesnt make sense to have any sort of dynamic code return a hard-coded value.

Reply:
Ah ok. I made it a property because its abstracted in InputMixIn and defined in both forward pass and data handling. I added an underscore :)

Reply:
Lol why is it a property anywhere though??? Why wouldn't we just set `self._worker_attrs = [...]` in all of the init's? Hardcoded properties don't make sense - abstracted upstream or not.